velox/type/Timestamp.cpp

```diff
@@ -66,7 +66,7 @@ void Timestamp::toGMT(const tz::TimeZone& zone) {
     // Invalid argument means we hit a conversion not supported by
     // external/date. Need to throw a RuntimeError so that try() statements do
     // not suppress it.
-    VELOX_FAIL(e.what());
+    VELOX_FAIL_UNSUPPORTED_INPUT_UNCATCHABLE(e.what());
```

Comment:
Could you share more context about this fix? Thanks.

Reply:
@rui-mo, this is for fixing a fuzzer failure: Unable to convert timezone 'America/Los_Angeles' past xxx, which means the tested input is beyond the supported time range.